models/yang/acme-system.yang

```diff
@@ -0,0 +1,50 @@
+     module acme-system {
```

Comment:
Why is this being added?

Reply:
I have deleted the file from the branch as it's not needed.